### ISSUE TITLE
v3.3.0 breaks Safari on BrowserStack

### ISSUE THREAD

**Issue by rodneyrehm** [theintern/intern#657]
The following Safari configuration on BrowserStack works fine with 3.2.3, but fails with 3.3.0:

```js
{
  browser: 'Safari',
  browser_version: '9.0',
  os: 'OS X',
  os_version: 'El Capitan',
  platform: 'MAC',
  browserName: 'Safari 9',
  'browserstack.safari.driver': '2.48',
}
```

Here's the console output for the failing run:

```
➜  npm run test:browserstack
Listening on 0.0.0.0:9000
Tunnel started
Suite Safari 9 on MAC FAILED
UnknownError: [POST https://(redacted)@hub.browserstack.com:443/wd/hub/session/<session>/url / {"url":"about:blank"}] undefined is not an object (evaluating 'a.postMessage') (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4 milliseconds
Build info: version: '2.48.2', revision: '41bccdd', time: '2015-10-09 19:59:12'
System info: host: 'mac-185-44-131-6.browserstack.com', ip: '185.44.131.6', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.safari.SafariDriver
Capabilities [{browserName=safari, takesScreenshot=true, javascriptEnabled=true, version=9.1, cssSelectorsEnabled=true, platform=MAC, secureSsl=true}]
Session ID: null
  at runRequest  <node_modules/intern/node_modules/leadfoot/Session.js:88:40>
  at <node_modules/intern/node_modules/leadfoot/Session.js:109:39>
  at new Promise  <node_modules/intern/node_modules/dojo/Promise.ts:411:3>
  at ProxiedSession._post  <node_modules/intern/node_modules/leadfoot/Session.js:63:10>
  at ProxiedSession.Session.get  <node_modules/intern/node_modules/leadfoot/Session.js:580:15>
  at ProxiedSession.lang.mixin.get  <node_modules/intern/lib/ProxiedSession.js:106:22>
  at <node_modules/intern/node_modules/leadfoot/Server.js:1110:20>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>

TOTAL: tested 0 platforms, 0/0 tests failed; fatal error occurred
Error: Run failed due to one or more suite errors
  at emitLocalCoverage  <node_modules/intern/lib/executors/Executor.js:353:18>
  at finishSuite  <node_modules/intern/lib/executors/Executor.js:370:15>
  at <node_modules/intern/lib/executors/Executor.js:378:8>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:420:9>
  at process._tickCallback  <node.js:349:13>
```

**Comment by rodneyrehm** [comment]
Making intern 3.3.0 use [leadfoot 1.6.10](https://github.com/theintern/leadfoot/releases/tag/1.6.10) solves the problem. However I can't find any obvious issue in the [diff to 1.6.11](https://github.com/theintern/leadfoot/compare/1.6.10...1.6.11).

**Comment by jason0x43** [comment]
The problem is with a new feature test added in discoverServerFeatures. Most feature tests (including the new one) will break Safari, and I didn't add a guard around it.

**Comment by jason0x43** [comment]
Fixed in theintern/leadfoot@b8fe0df20062c2cb386c144e946a38a1a1624df8

**Comment by jason0x43** [comment]
New release shortly...
